src/main/java/net/minecraftforge/oredict/ShapedOreRecipe.java

```diff
@@ -44,6 +47,7 @@
     protected int width = 0;
     protected int height = 0;
     protected boolean mirrored = true;
+    protected Map<Integer, FluidStack> fluidMap = new HashMap<Integer, FluidStack>();
```

Comment:
@HenryLoenwind Some mods enlarged crafting grid. Your suggestion may bring a break to those mods.